ptf/sai_base_test.py

```diff
@@ -276,7 +276,7 @@ def check_cpu_port_hdl(self):
                 index=True,
                 parent_scheduler_node=True)
             self.assertEqual(queue, q_attr['index'])
-            self.assertEqual(self.cpu_port_hdl, q_attr['port'])
+            self.assertEqual(self.cpu_port_hdl, q_attr['port']-1)
```

Comment:
In broadcom platform, the q_attr["port"] is not equals to cpu_port_hdl.

Reply:
So some platform(s) will pass the original assertion? If so, then the assertion should be platform specific.

Reply:
change to different assertion based on platform